web/webkit/src/main/scala/net/liftweb/http/Req.scala

```diff
@@ -284,14 +284,14 @@ FileParamHolder(name, mimeType, fileName)
  * @param localFile The local copy of the uploaded file
  */
 class OnDiskFileParamHolder(override val name: String, override val mimeType: String,
-                            override val fileName: String, val localFile: File) extends
+                            override val fileName: String, val localFile: Path) extends
```

Comment:
Whelp, I missed this guy. This will break all consumers of `OnDiskFileParamHolder`. Is there a way to get from `Path` to `File`? If not, I suggest we revert the changes to `OnDiskFileParamHolder`. If yes, I suggest we rename this parameter `localPath` and provide a `localFile` accessor that returns a `File` as existing code will expect. We can mark that one deprecated if we really want to.